src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml

```diff
@@ -334,8 +334,17 @@
         <div id="review-right" class="col-10 rounded-1 border">
             <table class="code-window">
                 <tbody>
+                    @{
+                        var commentThreadNumber = 0;
+                        ViewData["CommentThreadCount"] = @Model.Lines.Where(l => l.CommentThread != null).Count();

```

Comment:
nit: why not set this after the for loop below since we are going to be counting all the lines with comments anyway.

Reply:
Unfortunately, that does not work. I suppose it has to do with how ASP.Net  works. If I set it after the loop, it is null on the other side

Reply:
That is strange - I wouldn't expect it to behave differently than CommentThreadNumber